pkg/kubeadm/master/manifests.go

```diff
@@ -216,7 +216,8 @@ func getComponentCommand(component string, s *kubeadmapi.KubeadmConfig) (command
 			"--cluster-signing-cert-file=" + pki_dir + "/ca.pem",
 			"--cluster-signing-key-file=" + pki_dir + "/ca-key.pem",
 			"--insecure-experimental-approve-all-kubelet-csrs-for-group=system:kubelet-bootstrap",
-			"--cluster-cidr=" + s.InitFlags.Services.CIDR.String(),
+			"--allocate-node-cidrs=true",
+			"--cluster-cidr=" + s.InitFlags.PodNetwork.CIDR.String(),
```

Comment:
I think this should be optional, it really depends on the choice of network and user has to mach network configuration to reflect this.